tenets/codelingo/effective-go/README.md

```diff
@@ -0,0 +1,5 @@
+# Effective Go
+
+Useful tenets from Effective Go for your Go repository.
+
+### Ideas
```

Comment:
Remove 'Ideas' heading since its empty. Feel free to fix this for other readmes :)